themes/Default/theme.ini

```diff
@@ -9,3 +9,7 @@ qss_MAC=OSX Dark.qss
 name=Lite
 qss=Lite.qss
 qss_MAC=OSX Lite.qss
+[auto]
+name=Auto
```

Comment:
I guess you added this "theme" as a placeholder dummy, right? Take a look at this:

https://github.com/mumble-voip/mumble/blob/70dea6077854d79d4cb60b3ed1e8f1f8f78963a5/src/mumble/LookConfig.cpp#L120-L121

There we add the "None" theme. We could just as easily add the "Auto" theme there, so we do not need to have a dummy theme in ``theme.ini``

Reply:
Originily i tried to add the option with "qcbTheme->addItem(tr("Auto"));" , but it didnt work right as i think it didnt save/load properly. I tried to fix it by changing LookConfig::reloadThemes ,LookConfig::load and LookConfig::save, but to no avail. So I had to resort to doing with putting fake theme in theme.ini .

Reply:
You are right just adding ``qcbTheme->addItem(tr("Auto"));`` does not seem to be enough. However, from a maintainability point of view and general structure of this sort of this, I would insist that we solve this programmatically. I think that means modification to ``Themes::getConfiguredStyle`` and possibly a custom ``ThemeMap`` object.

(Please note that I do not have the time currently to properly 100% think this through, so that's also why I am very glad that someone else is taking this feature request 😅)